lib/reek/cli/input.rb

```diff
@@ -1,7 +1,7 @@
 require 'reek/source'
 
 module Reek
-  module CLI
+  module Cli
```

Comment:
But "CLI" is an acronym, not a word. I have no hard feelings towards this but it seems a little strange to me...